packages/kbn-cases-components/src/tooltip/tooltip.test.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { I18nProvider } from '@kbn/i18n-react';
+
+import { Tooltip } from './tooltip';
+import { CaseStatuses } from '../status/types';
+import type { CaseTooltipProps } from './types';
+
+const elasticUser = {
+  fullName: 'Elastic User',
+  username: 'elastic',
+};
+
+const tooltipProps: CaseTooltipProps = {
+  title: 'Another horrible breach!!',
+  description: 'Demo case banana Issue',
+  createdAt: '2020-02-19T23:06:33.798Z',
+  createdBy: elasticUser,
+  totalComments: 1,
+  status: CaseStatuses.open,
+};
+
+const sampleText = 'This is a test span element!!';
+const TestSpan = () => <span data-test-subj="sample-span">{sampleText}</span>;
+
+describe('Tooltip', () => {
```

Comment:
nit: Maybe we can add a test for checking the title and a test for checking the description.